quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/cli/Picocli.java

```diff
@@ -788,9 +788,13 @@ private static String getDecoratedOptionDescription(PropertyMapper<?> mapper) {
             }).toList();
 
             var isStrictExpectedValues = mapper.getOption().isStrictExpectedValues();
-            var printableValues = String.join(", ", decoratedExpectedValues) + (!isStrictExpectedValues ? ", or a custom one" : "");
+            var isCaseInsensitiveExpectedValues = mapper.getOption().isCaseInsensitiveExpectedValues();
+            var printableValues = String.join(", ", decoratedExpectedValues) + (!isStrictExpectedValues ? ", or a custom one" : (isCaseSensitiveExpectedValues ? " (case insensitive)" : ""));
```

Comment:
```suggestion
            var printableValues = String.join(", ", decoratedExpectedValues) + (!isStrictExpectedValues ? ", or a custom one" : "");
```
@shawkins I think this should be removed as well.

Reply:
See the latest changes. The reason I initially went with appending case sensitive is because it adds complexity to the short error message. Things are aligned now, but we'll have even more code to handle this situation.